druid-shell/src/lib.rs

```diff
@@ -17,6 +17,8 @@
 pub use piet_common as piet;
 pub use piet_common::kurbo;
 
+pub use unicode_segmentation;
```

Comment:
Hmm. I'm not sure I want druid-shell to become a "batteries included" aggregator for libraries we want. It feels like this dependency is better in druid than druid-shell. We're doing this for kurbo and piet because those are necessary dependencies from druid-shell.

That said, the idea that druid depends on unicode-segmentation seems quite reasonable to me.